media-libs/noise-suppression-for-voice/files/noise-suppression-for-voice-1.10-Patch-out-use-of-stat64.patch

```diff
@@ -0,0 +1,42 @@
+From 6b2901929fa7d618bda890adc7ecc15cb18d5b01 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Kacper=20S=C5=82omi=C5=84ski?=
+ <kacper.slominski72@gmail.com>
+Date: Sun, 11 Aug 2024 23:31:25 +0200
+Subject: [PATCH 2/3] Patch out use of stat64
+
+Not available on musl, and we're better off with using
+-D_FILE_OFFSET_BITS=64 instead anyway.
+---
```

Comment:
@vimproved as the author of the linked upstream issue, thoughts?

Reply:
For this purpose it's fine, but for the upstream JUCE pull request, I chose to keep the use of stat64 since it's also used on iOS and I'm not sure if iOS handles _FILE_OFFSET_BITS like glibc does.

Reply:
In this patch I've simply dropped the `JUCE_LINUX ||` bit from the condition around `stat64`, so it's still around for the iOS code path.